tests/test_androidtv_async.py

```diff
@@ -93,6 +93,35 @@ async def test_start_intent(self):
                 getattr(self.atv._adb, self.ADB_ATTR).shell_cmd, "am start -a android.intent.action.VIEW -d TEST"
             )
 
+    @awaiter
```

Comment:
Can you please revert the changes in this file. The source code that you added is already covered by the sync test.